src/world/common/UnkFunc49.inc.c

```diff
@@ -0,0 +1,9 @@
+#include "common.h"
+#include "map.h"
+
+ApiStatus N(UnkFunc49)(Evt* script, s32 isInitialCall) {
+    script->varTable[5] = (cos_deg(-(f32) script->varTable[3] - 35.0) * 3.0) - 2.3;
+    script->varTable[6] = sin_deg(-(f32) script->varTable[3] - 35.0) * 5.0;
```

Comment:
I'd prefer you put varTable[3] in a temp and cast it once, if that matches

Reply:
Doesn't seem to match.